src/NovaAboutServiceProvider.php

```diff
@@ -95,4 +58,26 @@ public function register(): void
         // Add Nova Packages to the About Command
         NovaAbout::addPackage('rhyslees/nova-about');
     }
+
+    protected function validLicenseKey(): bool
+    {
+        $response = Nova::checkLicense();
+
+        return $response->status() == 204;
```

Comment:
It might be better to use the cache value here to avoid high request to the API.

```suggestion
        return Nova::checkLicenseValidity();
```